### DISCUSSION TITLE
One click workspace

### DISCUSSION THREAD

**Discussion by dm5n** [rxhanson/Rectangle#1017] [Ideas]
I'm imagining a scenario where a user can configure a set of apps to open together – eg Chrome, iTerm and Finder, preallocated to window locations eg Chrome on the right-half of screen, iTerm occupying left upper 2/3rds and Finder on lower left third.

Then with one click – the apps all open ready to go. Currently, very annoying in Ventura – opening and configuring windows positions of the same apps individually. Perhaps something like this already exists, help me out :)

**Comment by Aaron-Rumpler** [comment]
[Rectangle Pro](https://rectangleapp.com/pro) supports saved window layouts, ~~but I don't think it will open apps for you (I might be wrong on this though, @rxhanson can comment further)~~.

**Reply by Aaron-Rumpler** [reply]
Upon installing Rectangle Pro and doing some testing, it looks like I was wrong on that. Rectangle Pro can open apps upon using a saved layout, but the opened apps won't get put in the right place until you trigger the layout again.

**Reply by rxhanson** [reply]
Rectangle Pro will launch apps and will send the command to put the window in the right place, but there is a chance that certain apps will still be initializing and won't listen to the window command immediately. I do have plans to make it more robust, but it's not a trivial problem; too long of a delay between app launching and window command taking effect gives a feeling where windows are repositioning out from under you.
